lab-mike/test/shoe-test.js

```diff
@@ -0,0 +1,181 @@
+'use strict';
+
+require('dotenv').config({path: `${__dirname}/./.test.env`});
+const superagent = require('superagent');
+const expect = require('expect');
+const server = require('../lib/server.js');
+
+const API_URL = `http://localhost:${process.env.PORT}`;
+
+let tempShoe;
+
+describe('testing shoe routes', () => {
+  before(server.start);
+  after(server.stop);
+
+  describe('test POST /api/shoes', () => {
+    it('should respond with a shoe', () => {
+      return superagent.post(`${API_URL}/api/shoes`)
+        .send({
+          brand: 'nike',
+          color: 'red',
+          size: 13,
+        })
+        .then((res) => {
+          expect(res.status).toEqual(200);
+          expect(res.body._id).toExist();
+          expect(res.body.brand).toEqual('nike');
+          expect(res.body.color).toEqual('red');
+          expect(res.body.size).toEqual(13);
+          tempShoe = res.body;
+        });
+    });
+    it('should respond with a 400 validation failed', () => {
```

Comment:
Consider using the 'it' blocks a little more expressively. I really like that you're testing all of these variations. However, you can make running the tests more impressive you describe the test more so that the messages in the terminal really show how robustly your routes handle errors.
ex: it('without supplying a BRAND the server should respond with a 400 error')